IntegrationExample/AppDelegate.swift

```diff
@@ -9,27 +9,53 @@ import Foundation
 import MimiCoreKit
 import MimiUXKit
 import MimiSDK
+import Combine
 
 final class AppDelegate: NSObject, UIApplicationDelegate {
-    
+
+    internal let headphoneConnectivityController = PartnerHeadphoneConnectivityController()
+
```

Comment:
nit: remove extra line

Reply:
I think it's good to separate private from internal variables.